tests/subsys/debug/coredump/src/main.c

```diff
@@ -30,7 +30,6 @@ void func_3(uint32_t *addr)
 #if defined(CONFIG_BOARD_M2GL025_MIV) || \
 	defined(CONFIG_BOARD_HIFIVE1) || \
 	defined(CONFIG_BOARD_LONGAN_NANO) || \
```

Comment:
doc needs updating

Reply:
hmm which docs?

Reply:
<board>/doc/index.rst

Reply:
I've already updated boards index file for sipeed, is that what you mean?

Reply:
https://github.com/teslabs/zephyr/blob/gd32vf103-hwmv2/boards/sipeed/longan_nano/doc/index.rst this file, the build commands there will not work if you keep the name requiring the soc name in it, will work if you remove the soc name from the base board (preferred option)

Reply:
hmm `west build -b longan_nano` still works for me, the full name is only a Twister thing I guess.

Reply:
I'd honestly prefer twister to follow the same logic as `west build`, so that we do not have to worry on what we put on the board yaml files.

Reply:
See https://github.com/zephyrproject-rtos/zephyr/tree/collab-hwm/boards/raspberry_pi/rpi_pico for example, `rpi_pico` is base board, twister, west, cmake, etc. all use that and that is the full filename, `rpi_pico/rpi_2040/w` is the variant but needs the soc in it, so that is the board, twister, etc. name and filename is `rpi_pico_rpi_2040_w`

Reply:
see latest revision